### PR TITLE
Add multiple pages

### PR DESCRIPTION
**Pull request by UCSD-Ethan-Ha**
Added home, workouts, and goals page. Designed homepage but unsure if unit tests can run. Issue with "react-router-dom"

### CODE REVIEW

File: client/src/pages/GoalTrackerPage.tsx
Comment (edolosa) [inline client/src/pages/GoalTrackerPage.tsx:1]: Great job with the Page set up, but does this GoalTrackerPage need the useWorkouts() constants? It looks rather cluttered with things that aren't in use.

**Review by edolosa — CHANGES_REQUESTED** [review]
These pages look great, but just one minor thing in GoalTrackerPage.tsx, we have the constants and imports for useWorkouts but they aren't needed. Removing them can help declutter the file. 

Other than that, great work here Ethan :D

File: client/src/pages/GoalTrackerPage.tsx
Comment (UCSD-Ethan-Ha) [reply client/src/pages/GoalTrackerPage.tsx:1]: Forgot to remove that. I'll take care of that now ty!

**Review by edolosa — APPROVED** [review]
Immaculate

**Comment by UCSD-Ethan-Ha** [comment]
W

**Pull request merged**